tests/state_mgen_test.go

```diff
@@ -267,20 +274,70 @@ func (tm *testMatcherGen) stateTestsGen(w io.WriteCloser, writeCallback, skipCal
 
 			// Since we know that tests run with and without the snapshotter features are equivalent, either boolean state
 			// is valid and 'false' is arbitrary.
+			if test.json.Tx.GasPrice == nil {
+				test.json.Tx.GasPrice = big.NewInt(1)
+			}
```

Comment:
This might be able to be removed. Tests failed because they could not be applied without a gas price. But now I'm thinking that those tests are probably the EIP1559 ones, and because positive error handling is now a part of the test data structures, that this is a hacky workaround that we don't need anymore.